frontend/src/app/model/forms-data.model.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * @description
+ *    This model holds the all the interfaces related to forms data.
+ */
```

Comment:
Important: When you write a comment like this, then it will always show up when someone hovers the "FormsData" interface. But I guess you want this comment only to be visible within this file, right? In this case, it would make more sense not to use a `/**` comment, but rather a normal `/*` comment. 

So, in general: If you add a comment to a specific element (e.g. a class, interface, method, class field, ...), then use the `/**` comments. But if you write a "general" comment that is not tied to a specific element, then use the `/*` comment. 

You can think of it like this: "Do I want the comment to show up when someone hovers the method name, class field, whatever?" Then, use the `/**` comment. But if you don't want it to show up when someone hovers the name of the next element (e.g. because this is not directly referring to the item, but is rather an overall explanation), then use the `/*` one. 

I hope that is somewhat understandable? Sorry if I haven't explained it properly in the previous discussions.

You don't need to change it now - just keep it in mind for the future :)

Reply:
I understood that. I puposefull

> Looking really good! 👍 🎉
> 
> I added a few review comments, but if you have read them, you can simply merge this PR if you want to 👍
> 
> It would be nice if you could create an Issue where we list the things we still need to do. Stuff that comes to my mind (partially as you have already mentioned):
> 
> * Use SQLite so that we are sure data won't be lost
> * Find a way to use multiple tables
> * Implement error handling
> * Store the data once someone clicks on the "submit" button (I have only had a brief look at it, but currently it is stored before, am I right?)

I will file issues for better tracking.